### ISSUE TITLE
SIGTRAP IBGUserAttributesLogger (Crash)

### ISSUE THREAD

**Issue by SudoPlz** [Instabug/instabug-reactnative#109]
Hey there, I'm not sure what this is, but a lot of our users are receiving a crash because of this error.
Here's the stack-trace, maybe you can tell more than we can from this:

```
SIGTRAP
    /usr/lib/system/libdispatch.dylib __dispatch_barrier_sync_f_slow
    Frameworks/InstabugCore.framework/InstabugCore -[IBGUserAttributesLogger getUserAttributes]
    Frameworks/InstabugCore.framework/InstabugCore -[IBGSnapshotManager getSurveyConditionsWithCompletionHanlder:]
    Frameworks/Instabug.framework/Instabug -[InstabugSurveys showSurveyIfAvailable]
    Frameworks/CoreFoundation.framework/CoreFoundation ___CFNOTIFICATIONCENTER_IS_CALLING_OUT_TO_AN_OBSERVER__
    Frameworks/CoreFoundation.framework/CoreFoundation __CFXRegistrationPost
    Frameworks/CoreFoundation.framework/CoreFoundation ____CFXNotificationPost_block_invoke
    Frameworks/CoreFoundation.framework/CoreFoundation -[_CFXNotificationRegistrar find:object:observer:enumerator:]
    Frameworks/CoreFoundation.framework/CoreFoundation __CFXNotificationPost
    Frameworks/Foundation.framework/Foundation -[NSNotificationCenter postNotificationName:object:userInfo:]
    Frameworks/Foundation.framework/Foundation -[NSNotificationCenter postNotificationName:object:]
    Frameworks/InstabugCore.framework/InstabugCore __52-[IBGUserAttributesLogger setUserAttribute:withKey:]_block_invoke
    /usr/lib/system/libdispatch.dylib __dispatch_call_block_and_release
    /usr/lib/system/libdispatch.dylib __dispatch_queue_serial_drain
    /usr/lib/system/libdispatch.dylib __dispatch_queue_invoke
    /usr/lib/system/libdispatch.dylib __dispatch_queue_override_invoke
    /usr/lib/system/libdispatch.dylib __dispatch_root_queue_drain
    /usr/lib/system/libdispatch.dylib __dispatch_worker_thread3
    /usr/lib/system/libsystem_pthread.dylib __pthread_wqthread
```

"react-native": "0.51.0",
"instabug-reactnative": "2.1.2",
Device Apple iPad 4
iOS 10.3.3

**Comment by ZiadAyman** [comment]
This issue is fixed in our latest react-native version. 
Unlink, install the latest version and link again please @SudoPlz 

Check the upgrade file here:
https://github.com/Instabug/instabug-reactnative/blob/master/README.md#upgrading-guide

Let me know how is it going :)

**Comment by SudoPlz** [comment]
You mean it will be resolved if I upgrade to 2.2.2? I'll try that, thank you.

**Comment by ZiadAyman** [comment]
Exactly. Give it a try and let me know if you still experience the same signal.
